packages/devtools/lib/src/ui/analytics.dart

```diff
@@ -315,30 +314,19 @@ Future<void> computeUserApplicationCustomGTagData() async {
 
   if (isFlutter && !isProfile) {
     // Compute the Flutter platform for the user's running application.
-    final VmService vmService = serviceManager.service;
-    final io = EvalOnDartLibrary(['dart:io'], vmService);
-
-    // eval user's Platform for all possible values.
-    final android = await io.eval('Platform.isAndroid', isAlive: null);
-    final iOS = await io.eval('Platform.isIOS', isAlive: null);
-    final fuchsia = await io.eval('Platform.isFuchsia', isAlive: null);
-    final linux = await io.eval('Platform.isLinux', isAlive: null);
-    final macOS = await io.eval('Platform.isMacOS', isAlive: null);
-    final windows = await io.eval('Platform.isWindows', isAlive: null);
-
-    if (android.valueAsString == 'true') {
-      userPlatformType = platformTypeAndroid;
-    } else if (iOS.valueAsString == 'true') {
-      userPlatformType = platformTypeIOS;
-    } else if (fuchsia.valueAsString == 'true') {
-      userPlatformType = platformTypeFuchsia;
-    } else if (linux.valueAsString == 'true') {
-      userPlatformType = platformTypeLinux;
-    } else if (macOS.valueAsString == 'true') {
-      userPlatformType = platformTypeMac;
-    } else if (windows.valueAsString == 'true') {
-      userPlatformType = platformTypeWindows;
-    }
+    final vmService = serviceManager.service;
+
+    // Return values from operatingSystem can be:
+    //    android
+    //    linux
+    //    ios
+    //    macos
+    //    windows
+    //    fuchsia
+    userPlatformType =
+        (await vmService.isProtocolVersionLessThan(major: 3, minor: 24))
+            ? 'unknown'
+            : (await vmService.getVM()).operatingSystem;
```

Comment:
instead of calling `getVm()` you can just use the value we cache upon vm service connection: `serviceManager.vm`.

If you do this, you can also remove the local var `vmService` and instead write `serviceManager.service.isProtocolVersionLessThan(...)`

Reply:
Done.